app/helpers/application_helper.rb

```diff
@@ -290,6 +290,7 @@ def admin_config_description(content)
   end
 
   def role_display_name(role)
+    # TODO: [@jacobherrington] remove the check in this method
```

Comment:
It might be helpful to reference here what will allow this to be removed. For example, once this PR has been merged and deployed. That way if we do accidentally forget about it when someone comes along they have the necessary context to know when and under what circumstances this is safe to remove.